apache/nifi
# NIFI-11287: detect dependent properties when the property it depends on references a parameter (UI)

!-- Licensed to the Apache Software Foundation (ASF) under one or more -->












# Summary

[NIFI-11287](https://issues.apache.org/jira/browse/NIFI-11287) (UI)
To test:
- add the Compress Content processor to the canvas
- for property "Mode", select between "compress" and "decompress". You should see the "Compress Level" dependent property show and hide
- create and add a parameter context with a parameter that has a value of "decompress" 
- back in the processor configurations, configure the "Mode" property with the new parameter. The "Compress Level" property should hide. 

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking


### Pull Request Tracking


### Pull Request Formatting


# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

  - [x] JDK 11
  - [ ] JDK 17

### Licensing


### Documentation
